lib/export/project_data/sql.rb

```diff
@@ -149,7 +149,8 @@ def self.export_users(io, project)
         created_by_id: user.created_by_id || 'NULL',
         updated_by_id: user.updated_by_id || 'NULL',
         is_administrator: user.is_administrator || 'NULL',
-        hub_tab_order: "'{#{conn.escape_string(user.hub_tab_order.join(','))}}'"
+        hub_tab_order: "'{#{conn.escape_string(user.hub_tab_order.join(','))}}'",
+        preferences: %['"#{conn.escape_string(JSON.generate(user.preferences)).gsub('"', '\"')}"']
```

Comment:
Have you tried `preferences: "'#{conn.escape_string(JSON.generate(user.preferences))}'"`? I think `'` is the char that needs to be escaped, not `"`.

```ruby
3.3.1 :017 > puts ActiveRecord::Base.connection.raw_connection.escape_string("\"hello\" 'bye'")
"hello" ''bye''
```